pl/README.md

```diff
@@ -0,0 +1,39 @@
+# Uwagi dotyczące tłumaczenia
+
+Celem tego pliku jest zdefiniowanie pewnych wytycznych dla ludzi, którzy chcą uczestniczyć w pracach nad tłumaczeniem tego projektu. Jak wiadomo, tłumaczenie z jednego języka na drugi bywa niezwykle złożonym zadaniem, przepełnionym elementami subiektywnego podejścia do tematu. Z tego też powodu wydaje się być niezbędnym, by wyznaczać pewne zalecenia, które będą powszechnie uznane za obowiązujące każdego, kto podejmuje się tłumaczenia. Przykładem takich reguł mogą być zasady dotyczące tłumaczenia określonych wyrażeń, bądź wyznaczenie takich wyrażeń, których z uwagi na ich kontekst czy znaczenie nie powinno się tłumaczyć.
+
+## Zalecenia
+
+Obecnie brak
+
+## Powszechnie uznane tłumaczenia
+
+| Angielski                    | Polski                  |
+|------------------------------|-------------------------|
+| string                       | ciąg znaków / łańcuch   |
+| directive                    | dyrektywa               |
+| magic number                 | liczba magiczna (?)     |
+| array                        | tablica                 |
+| list (typ danych)            | lista (?)               |
+| map (typ danych)             | mapa (?)                |
+| nesting                      | zagnieżdżanie           |
+| namespace                    | przestrzeń nazw (?)     |
+| whitespaces                  | znaki niedrukowalne (?) |
+
```

Comment:
I'd add a comment to always use original word in case of doubt, as they are kind of lingua Franca. This guide is for developers, doctors are supposed to know Latin word for my medicine anyway.